index.bs

```diff
@@ -3076,11 +3076,11 @@ or [=authenticatorGetAssertion=] operation currently in progress.
 
 Authenticators may be required to store arbitrary strings chosen by a [=[RP]=], for example the {{PublicKeyCredentialEntity/name}}, {{PublicKeyCredentialUserEntity/displayName}}, and {{PublicKeyCredentialEntity/icon}} members in a {{PublicKeyCredentialUserEntity}}. Each will have some accommodation for the potentially limited resources available to an [=authenticator=].
 
-If that accommodation is that [=authenticators=] may truncate a value after a specific number of bytes then that MAY be done without reference to any string encoding. For example, an authenticator may truncate a {{PublicKeyCredentialEntity/name}} and leave a partial UTF-8 sequence at the end. If the authenticator is using [[!FIDO-CTAP]] then future messages from the authenticator may contain invalid CBOR since the value is typed as a CBOR string and thus is required to be valid UTF-8.
+If that accommodation is that [=authenticators=] may truncate a value after a specific number of bytes, the authenticator SHOULD take care to do so on character boundaries, although the authenticator MAY truncate without reference to any string encoding. For example, an authenticator could truncate a {{PublicKeyCredentialEntity/name}} and leave a partial UTF-8 sequence at the end. If the authenticator is using [[!FIDO-CTAP]] then future messages from the authenticator may contain invalid CBOR since the value is typed as a CBOR string and thus is required to be valid UTF-8.
```

Comment:
I'd replace "string encoding" with "character encoding".

Reply:
Done in 12763df.